GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -443,11 +456,43 @@ + (GIDSignIn *)sharedInstance {
   static dispatch_once_t once;
   static GIDSignIn *sharedInstance;
   dispatch_once(&once, ^{
-    sharedInstance = [[self alloc] initPrivate];
+    GTMKeychainStore *keychainStore =
+        [[GTMKeychainStore alloc] initWithItemName:kGTMAppAuthKeychainName];
+    sharedInstance = [[self alloc] initWithKeychainStore:keychainStore];
   });
   return sharedInstance;
 }
 
+#pragma mark - Configuring and pre-warming
+
+#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
+- (void)configureWithCompletion:(nullable void (^)(NSError * _Nullable))completion {
+  @synchronized(self) {
+    // This check helps to avoid needing to set `_appCheck` in an initializer called in
+    // `+[GIDSignIn sharedInstance]`. Also defers to the fakes for `_appCheck` set in tests.
+    if (!_appCheck) {
+      _appCheck = [[GIDAppCheck alloc] initWithAppCheckTokenFetcher:nil];
+    }
```

Comment:
IIUC, this is a temporary solution before we can put GIDAppCheck's initialization into the initializer of `GIDSignIn`. If so, can we leave a comment here to make it clear that this is temporary?

Reply:
https://github.com/google/GoogleSignIn-iOS/issues/322